src/Taylor1.jl

```diff
@@ -661,79 +883,20 @@ end
 
 ## Returns de n-th derivative of a series expansion
```

Comment:
This block (lines 884-894) is repeated and corresponds to the case `derivative(n:Int, a::Taylor1)`; please delete.